distributed/comm/tcp.py

```diff
@@ -193,9 +193,14 @@ async def read(self, deserializers=None):
             frames_nbytes = await stream.read_bytes(fmt_size)
             (frames_nbytes,) = struct.unpack(fmt, frames_nbytes)
 
-            frames = bytearray(frames_nbytes)
-            n = await stream.read_into(frames)
-            assert n == frames_nbytes, (n, frames_nbytes)
+            frames = memoryview(bytearray(frames_nbytes))
+            # Workaround for OpenSSL 1.0.2 (can drop with OpenSSL 1.1.1)
+            MAX = 1_000
```

Comment:
Am a little fuzzy on what this number should be. Somewhere this should be defined. Once we figure that out can replace with a better value and name. Can also define this as a global constant.